.travis-ocaml.sh

```diff
@@ -142,6 +142,7 @@ install_on_linux () {
         install_ppa avsm/ocaml42+opam12 ;;
     4.02,2.0.0)
         OCAML_FULL_VERSION=4.02.3
+        OPAM_SWITCH=${OPAM_SWITCH:-ocaml-base-compiler.4.02.3}
```

Comment:
I do not think this is a lasting solution: when the system com will change, this will need to be updated.  Alternative solution: https://github.com/ocaml/ocaml-ci-scripts/pull/263